src/models/constructor-items/common.ts

```diff
@@ -156,6 +156,8 @@ export interface LinkProps extends Stylable {
     target?: string;
     metrikaGoals?: MetrikaGoal;
     pixelEvents?: ButtonPixel;
+    // discuss how to be with this component
+    context?: string;
```

Comment:
we don't need this param till we use `useContext`

Reply:
ok